tests/tests_template.py

```diff
@@ -0,0 +1,10 @@
+import unittest
+
+
+class TemplateTest(unittest.TestCase):
```

Comment:
I guess this one is just for you

Reply:
Hmmmm good question weather to commit this or not, is that what you're saying?

Reply:
I'm kinda thinking that it is going to help someone developing in future cases, cause it helps him write tests faster